src/ProjectNeon/Assets/Features/Battle/Cards/EnemyCardSelection.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyCardSelection : MonoBehaviour
```

Comment:
This behavior isn't triggered anywhere. 

We should be able to witness this happening in the BattleScene.

Let's have this behavior happen once `TurnStarted` occurs.

`TurnStarted` can happen once `BattleSetupFinished` occurs.

Reply:
@SilasReinagel I've wired it all with events, please take a look. It was a little to understand, so I've left a puzzle for improve the documentation of `EventOrchestrator`